notebooks/integrations/integration-sklearn.ipynb

```diff
@@ -121,22 +121,24 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "experiment 41b6afda-430d-423f-8d08-0b5ab773339c\n",
-      "parameters: [('scaler__copy', True), ('scaler__with_mean', True), ('scaler__with_std', True), ('svc__break_ties', False), ('svc__C', 1.0), ('svc__cache_size', 200), ('svc__class_weight', None), ('svc__coef0', 0.0), ('svc__decision_function_shape', 'ovr'), ('svc__degree', 3), ('svc__gamma', 'scale'), ('svc__kernel', 'rbf'), ('svc__max_iter', -1), ('svc__probability', False), ('svc__random_state', None), ('svc__shrinking', True), ('svc__tol', 0.001), ('svc__verbose', False)]\n",
+      "experiment c935e31b-33bc-4982-9342-9c1fba69a1d8\n",
+      "parameters: [('scaler__copy', True), ('scaler__with_mean', True), ('scaler__with_std', True), ('svc__C', 1.0), ('svc__break_ties', False), ('svc__cache_size', 200), ('svc__class_weight', None), ('svc__coef0', 0.0), ('svc__decision_function_shape', 'ovr'), ('svc__degree', 3), ('svc__gamma', 'scale'), ('svc__kernel', 'rbf'), ('svc__max_iter', -1), ('svc__probability', False), ('svc__random_state', None), ('svc__shrinking', True), ('svc__tol', 0.001), ('svc__verbose', False)]\n",
+      "metrics: []\n",
+      "experiment d8098eee-0e9a-4d64-9c80-45464b9c600b\n",
+      "parameters: []\n",
       "metrics: [('score', 0.88)]\n"
```

Comment:
I think you ran the cell above this one twice in the notebook you committed. can you re-run it and commit it with each cell only run once? I'd expect this cell to only output one experiment